src/_data/sidenav.yml

```diff
@@ -117,10 +116,8 @@
         - title: Widget catalog
           permalink: /development/ui/widgets
     - title: Data & backend
-      permalink: /development/data-and-backend
       children:
         - title: State management
-          permalink: /development/data-and-backend/state-mgmt
```

Comment:
Or this one

Reply:
Yep! https://docs.flutter.dev/development/data-and-backend/state-mgmt